modules/client-configuration/pages/configuration-basics.adoc

```diff
@@ -0,0 +1,19 @@
+[[configuration.basics]]
+= Configuration Basics
+
+{productname} requires a number of steps to prepare the environment for clients registration before a wide range of its operations can be utilised.
+
+This section contains summary of the initial configuration steps that are necessary to support environment operations following successful {productname} installation and setting up.
+
+
```

Comment:
If we have several "For more info" statements in a row, use the itemized list layout.